overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -25810,7 +25810,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Basic §aFlux Capacitors§r hold a whopping lot of energy - §a10 million RF§r - and are useful for keeping your §aRF-powered§r equipment and tools charged.\n\nPut one into your newly crafted §3CEF§r to charge it. You can then put it in your inventory to charge your items with §aRF§r, or you can leave it in the CEF to act as a §a2.5 million EU§r battery!\n\nFlux Capacitor is a §bBauble§r, open your inventory and look for the Baubles button, click it to reveal the additional bauble slots menu and put it into any slot you wish.\n\nFinally you can charge up all those depleted §6Dark Boots§r and §6The Ender§r swords you\u0027ve looted.",
+          "desc:8": "Basic §aFlux Capacitors§r hold a whopping lot of energy - §a10 million RF§r - and are useful for keeping your §aRF-powered§r equipment and tools charged.\n\nPut one into your newly crafted §3CEF§r to charge it. You can then put it in your inventory to charge your items with §aRF§r, or you can leave it in the CEF to act as a §a2.5 million EU§r battery!\n\nFlux Capacitor is a §bBauble§r, open your inventory and look for the Baubles button, click it to reveal the additional bauble slots menu and put it into any slot you wish.\n\nFinally, you can charge up all those depleted §6Dark Boots§r and §6The Ender§r swords you\u0027ve looted.",
```

Comment:
While a comma typically follows the use of "Finally" when used to denote the last item in a sequence of points (as in "First, .... Second, .... Finally, ..."), this usage is reflective of exasperation (i.e. "ugh, FINALLY I can do X") and the pause doesn't really fit.

English is weird. See also [CMOS FAQ](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0001.html) which touches on this sort of flexibility of comma rules.

Reply:
Seems that now I can become even better. Thanks for FAQ. Reverted.